src/docs/metrics/social-volume/index.md

```diff
@@ -1,197 +1,164 @@
+[Social Data latency](/metrics/details/latency#social-data-latency)
 
-### Measuring Unit
+---
 
-Relative number from 0 to 1 (or %, from 0 to 100).
+## Available Assets
 
-### Frequency
+Available for [these
+assets](<https://api.santiment.net/graphiql?variables=&query=%7B%0A%20%20getMetric(metric%3A%20%22social_volume_total%22)%20%7B%0A%20%20%20%20metadata%20%7B%0A%20%20%20%20%20%20availableSlugs%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A>)
 
-Same as [**Social Volume**](#social-volume).
+> Note: `social_volume_total` metric and all metrics for a specific source are
+> available for the same set of assets.
 
-### Latency
+---
 
-Same as [**Social Volume**](#social-volume).
+## Sanbase
 
-### Available Assets
+Combined Social Volume from all sources for an asset can be seen on a [project's page](https://tinyurl.com/uqemh89).
 
-Same as [**Social Volume**](#social-volume).
+Social Volume for arbitrary search terms can be viewed from the [trends page](https://app.santiment.net/labs/trends)
+![trends page](trends-page.png)
 
-By default the **Social Dominance** for a given asset is calculated against the top 50 assets by market capitalization.
+The combined social volume from all sources is displayed. From the bottom of the page social volume for a specific source can be displayed, too.
+![social volume search term](social-volume-search-term.png)
 
-### How to Access
+## SanAPI
 
-#### [Sanbase](https://app.santiment.net)
+Available under the `social_volume_total` and `social_volume_total_<source>`
+names, where the available sources are:
 
-The metric is available on Sanbase on our charts **for any selected asset**:
+- telegram
+- reddit
+- discord
+- professional_traders_chat
+- total (combines all sources)
 
-<iframe frameborder="0" height="340" src="https://app.santiment.net/chart?from=2019-06-02T21%3A00%3A00.000Z&interval=12h&isShowAnomalies=true&metrics=historyPrice,socialDominance&scale=auto&slug=santiment&title=Santiment%20%28SAN%29&to=2019-12-03T21%3A00%3A00.000Z&viewOnly=true"></iframe>
+### Social Volume for an asset
 
-#### [Sanbase Graphs](https://graphs.santiment.net/social)
+```graphql
+{
+  getMetric(metric: "social_volume_total") {
+    timeseriesData(
+      selector: { slug: "santiment" }
+      from: "2020-01-01T00:00:00Z"
+      to: "2020-01-07T00:00:00Z"
+      interval: "1d"
+    ) {
+      datetime
+      value
+    }
+  }
+}
+```
 
-The metric is available **for any selected asset**.
+**[Run in
+Explorer](<https://api.santiment.net/graphiql?query=%7B%0A%20%20getMetric(metric%3A%20%22social_volume_total%22)%20%7B%0A%20%20%20%20timeseriesData(%0A%20%20%20%20%20%20selector%3A%20%7B%20slug%3A%20%22santiment%22%20%7D%0A%20%20%20%20%20%20from%3A%20%222020-01-01T00%3A00%3A00Z%22%0A%20%20%20%20%20%20to%3A%20%222020-01-07T00%3A00%3A00Z%22%0A%20%20%20%20%20%20interval%3A%20%221d%22%0A%20%20%20%20)%20%7B%0A%20%20%20%20%20%20datetime%0A%20%20%20%20%20%20value%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D>)**
 
-### Availability
```

Comment:
This section seems important to me.

Reply:
I've removed it from all metrics as we have changes in the available plans and there are ongoing changes of moving metrics to different plans. It can be reintroduced when we have final decisions on these topics.